primp/primp.pyi

```diff
@@ -0,0 +1,278 @@
+from typing import Dict, Optional, Tuple, Union, Literal
+
+IMPERSONATE = Literal[
```

Comment:
This list is incomplete

Reply:
Added missing firefox variants

Reply:
Something else has changed, check README

Reply:
made the changes